src/docs/product/releases/suspect-commits/index.mdx

```diff
@@ -82,31 +86,111 @@ The following example associates commits (or refs) between `from` and `to` with
 sentry-cli releases set-commits --commit "my-repo@from..to" $VERSION
 ```
 
+Alternatively, you can use the `--local` flag to enable this behavior by default.
```

Comment:
@kamilogorek - I've made an edit below using the wording you've used, but I'm wondering, is the behavior we're talking about what's described on line 79?

```suggestion
Alternatively, you can use the `--local` flag so that, by default, we fetch all commits between the `head` (the latest commit available locally) and the most recent one that was used for the previous Sentry release.
```

Reply:
Kinda, but not quiet the same. On line 79 it states "then the `--auto` flag will automatically use the git tree of your local repo". That is not exactly how it behaves.

`--auto` will ask local repository for the address of the remote git server. Then it will ask this server for the state of commits and use that history to determine which commits to associate with the release.
Where `--local` will not reach out to any remote servers, but rather use local history for that detecion.

Reply:
@imatwawana, I think the issue is that we need to fix the spot earlier where it talks about `--auto`, and maybe ditch this line all together. Before I propose a revision to that earlier paragraph, though... @kamilogorek, when you're using `--auto` and it's pulling the commits from the remote repo - how does it pull those commits if you don't have a GH/BB/GL integration? I'm asking because the paragraph I'm proposing to change says

> **_If you don't have a repo-based integration associated with your Sentry organization_**, then the `--auto` flag blah blah blah.

Reply:
@lobsterkatie & @kamilogorek, Nisanthan is out sick and it's been almost 20 days since I opened this PR. I'd like to leave the language as is (especially since it's not clear that this has been a blocker for folks), merge this PR, and open a new PR to just deal with that language change.